### DISCUSSION TITLE
Does bat use Bubbles?

### DISCUSSION THREAD

**Discussion by cloudk1d** [sharkdp/bat#2252] [Q&A] [answered]
I am working on a command line utility and wondering if the "viewport" that bat uses to display a file makes use of [Bubbles](https://github.com/charmbracelet/bubbles)? Thanks!

**Comment by keith-hall** [comment, marked as answer]
Nope, just your standard `PAGER` application, generally defined to be `less` by default on most systems. N. B. Looks like Bubbles is written in Go, whereas `bat` is a Rust utility.
